_data-prepper/pipelines/configuration/processors/routes.md

```diff
@@ -10,10 +10,58 @@ nav_order: 90
 
 Routes define conditions that can be used in sinks for conditional routing. Routes are specified at the same level as processors and sinks under the name `route` and consist of a list of key-value pairs, where the key is the name of a route and the value is an OpenSearch Data Prepper expression representing the routing condition.
 
-<!---## Configuration
+## Configuration
```

Comment:
Thanks @Naarcha-AWS for working on this. Data Prepper doesn't have a `routes` processor at all.

Our routes concept is defined in the top-level of the pipeline. It is documented here:

https://docs.opensearch.org/docs/latest/data-prepper/pipelines/pipelines/#conditional-routing

Do you think this needs to be updated?

Reply:
I think the move would be to remove this in order to avoid confusion. We can redirect to the conditional routing.